tests/accounts/test_integration.py

```diff
@@ -52,11 +52,32 @@ def test_existing_account_is_returned(self):
                          self.account2['accountInfo']['accountNumber'])
 
 
+class TestAccountCreationWithUserTokenPassed(AccountTestCase):
+    def setUp(self):
+        service = SitewitService()
+        site_id = uuid.uuid4()
+
+        response = service.create_account(
+            site_id, self.url, self.user_name, self.user_email,
+            self.currency, self.country_code)
+
+        self.user_token = response['userInfo']['token']
+
+        self.response = service.create_account(
+            uuid.uuid4(), 'https://foo.bar', self.user_name,
+            '{}@yola.com'.format(uuid.uuid4()),
+            self.currency, self.country_code, self.user_token)
+
+    def test_creates_new_account_for_the_given_user(self):
+        self.assertIsNotNone(self.user_token)
+        self.assertEqual(self.user_token, self.response['userInfo']['token'])
```

Comment:
I think what @RayeN did is reasonable. Since this is a public client for SiteWit's API, it makes sense to be consistent with their terminology and assume that users are at least somewhat familiar with that terminology.